lib/bridge/IrcBridge.js

```diff
@@ -20,7 +20,8 @@ var BridgeRequest = require("../models/BridgeRequest");
 var stats = require("../config/stats");
 var DataStore = require("../DataStore");
 var log = require("../logging").get("IrcBridge");
-const { Bridge, MatrixUser, PrometheusMetrics, Logging } = require("matrix-appservice-bridge");
+const { Bridge, MatrixUser, MatrixRoom,
+    PrometheusMetrics, Logging } = require("matrix-appservice-bridge");
```

Comment:
```suggestion
    PrometheusMetrics,
    Logging,
} = require("matrix-appservice-bridge");
```

/me guesses if this is how it works in JS land.

Reply:
yis